modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -78,23 +78,12 @@ public static long addListTypeDefinition(
 					userId, objectField.getSystem());
 		}
 
-		ObjectFieldSetting[] objectFieldSettings = ArrayUtil.filter(
+		ObjectFieldSetting[] stateFlowSettings = ArrayUtil.filter(
```

Comment:
I think it is better to rename this variable to "stateFlowObjectFieldSettings". What do you think?

Reply:
Done!! :white_check_mark: